.atom-build.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Any particular reason? I am only curious why it's harmful, given that the repo already has `travis.yml